bentoml/cli/__init__.py

```diff
@@ -20,6 +20,7 @@
 import os
 import json
 import click
+import click_completion
```

Comment:
Thanks for the contribution @lintingzhen!

The CI tests are actually failing because of missing this library in a test environment.

Could you make it an optional dependency?
```
try:
  import click_completion
except ImportError:
   click_completion.init()
```
 and in the install command:
```
```
try:
  import click_completion
except ImportError:
  echo.error("'click_completion' is required for BentoML auto-completion, install it with `pip install click_completion`")
```
That way we don't need to add it to setup.py as a hard requirement and user can

Reply:
How to process `click_completion.DocumentedChoice` in argument?
```
    @bentoml_cli.command(
        help="Install shell command completion",
        short_help="Install shell command completion",
    )
    @click.option('--append/--overwrite',
                  help="Append the completion code to the file",
                  default=None)
    @click.argument('shell',
                    required=False,
                    type=click_completion.DocumentedChoice(click_completion.core.shells)
                    )
    @click.argument('path', required=False)
    def install_completion(append, shell, path):
```